e2etests/features/step_definitions/common.rb

```diff
@@ -1,7 +1,5 @@
-# frozen_string_literal:true
-
-Given(/^"([^"]*)" user$/) do |user|
-  user_access(user)
+Given(/the site is visited/) do
+  visit(TARGET_URL)
```

Comment:
This should really be in a method.  Where it could be re-used and potentially have related related code for different events.

Reply:
I disagree

It is as simple as it can be now.  Some English and a step which just visits the page

Adding an extra method call just complicates the code without a good reason

If a concrete reason to reuse it comes up, then a method can be added